scripts/quarantine.yaml

```diff
@@ -45,6 +45,8 @@
     - sample.matter.window_cover.debug
     - sample.matter.window_cover.release
     - sample.matter.window_cover.smp_dfu
+    - applications.asset_tracker_v2.nrf7002ek_wifi-debug
```

Comment:
According to the comment this list is only for Matter samples and to limit CI resources, that is not for tests that tend to fail. Please create a new list and describe the reason why it is in quarantine.